bci_tester/data.py

```diff
@@ -727,6 +727,12 @@ def create_BCI(
         available_versions=pg_versions,
         forwarded_ports=[PortForwarding(container_port=5432)],
         extra_environment_variables={"POSTGRES_PASSWORD": POSTGRES_PASSWORD},
+        # FIXME: https://github.com/SUSE/BCI-tests/issues/647
```

Comment:
please remove FIXME, that just lowers the pylint scoring and somebody else needs to compensate for that.

Reply:
Removed it.